.travis.yml

```diff
@@ -5,4 +5,5 @@ before_script:
   - npm install -g mocha
 node_js:
   - "0.12"
+sudo: required
```

Comment:
this sudo required can be removed. I was trying to allow sudo to see if this was causing CI failure.